rxbinding/src/androidTest/java/com/jakewharton/rxbinding/view/RxViewTest.java

```diff
@@ -260,6 +264,32 @@
     o.assertNoMoreEvents();
   }
 
+  @TargetApi(M)
+  @SdkSuppress(minSdkVersion = M)
+  @Test @UiThreadTest public void scrollChangeEvents() {
```

Comment:
I fixed it. Merging manually. Probably was that the view starts at 0,0 and ignores scrolls to the same position as the current one.